robot/basestation/static/js/elements/feed/camera.js

```diff
@@ -112,6 +112,39 @@ $(document).ready(() => {
     })
   })
 
+  $('#record-video-toggle').on('click', function (event) {
+    var el = $(this); // variable of button text
```

Comment:
Use `let` instead of `var`.  https://www.geeksforgeeks.org/difference-between-var-and-let-in-javascript/

Here it is a good instance of "Making good variable names to remove comments`

This line should be something `let recordingButton = $(this);`
There is no need to comment since it is obvious. You can look into the rest of your code to see what can be improved like this.